presto-native-execution/presto_cpp/main/runtime-metrics/RuntimeMetricsReporter.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "presto_cpp/main/http/HttpServer.h"
+#include "presto_cpp/main/runtime-metrics/prometheus-reporter/PrometheusStatsReporter.h"
```

Comment:
This include needs to guarded by #ifdef also.

```
#ifdef PRESTO_ENABLE_PROMETHEUS_REPORTER
#include "presto_cpp/main/runtime-metrics/prometheus-reporter/PrometheusStatsReporter.h"
#endif
```

Reply:
I will move it back to RuntimeStatsReporter.cpp. @amitkdutta are you sure about this? I though you are facing some issues on Meta side.

Reply:
Sorry didn't underestand. We want to include the header only when the macro is enabled.

Reply:
This is exactly another point why we should make that fetchMetrics() generic, such that API and implementations could be cleanly separated.